Documentation/teaching/labs/exercises-summary.hrst

```diff
@@ -1,44 +1,56 @@
+.. important::
 
-To solve exercises, you need to perform these steps:
+   To solve exercises, you need to perform these steps:
 
-   * prepare skeletons from templates
-   * build modules
-   * copy modules to the VM
-   * start the VM and test the module in the VM.
+      * prepare skeletons from templates
+      * build modules
+      * copy modules to the VM
+      * start the VM and test the module in the VM.
 
-The current lab name is |LAB_NAME|. See the exercises for the task name.
+   The current lab name is |LAB_NAME|. See the exercises for the task name.
 
-The skeleton code is generated from full source examples located in
-:file:`tools/labs/templates`. To solve the tasks, start by generating
-the skeleton code for a complete lab:
+   The skeleton code is generated from full source examples located in
+   :file:`tools/labs/templates`. To solve the tasks, start by generating
+   the skeleton code for a complete lab:
 
-.. code-block:: shell
+   .. code-block:: shell
 
```

Comment:
I don't understand why there is this indentation? As far as I remember starting directly at the beginning of a line is fine.

Reply:
Because I moved the important label from the outside file (the one that includes this file) inside this, and indented everything in the important block.

Reply:
@valx90 oh, ok. makes sense thanks!